Sources/Plasma/FeatureLib/pfPython/pyGUIControlListBox.cpp

```diff
@@ -241,21 +183,7 @@ class pfColorListElement : public pfGUIListText
 
         void    GetSize(plDynamicTextMap *textGen, uint16_t *width, uint16_t *height) override
         {
-            bool wemade_string = false;
-            wchar_t* thestring;
-            if ( fString1 && fString2 )
-            {
-                size_t length = wcslen( fString1 ) + wcslen( fString2 ) + 3;
-                thestring = new wchar_t[ length ];
-                swprintf( thestring, length, L"%s %s", fString1, fString2 );
-                wemade_string = true;
-            }
-            else if (fString1)
-                thestring = fString1;
-            else if (fString2)
-                thestring = fString2;
-            else
-                thestring = nullptr;
+            ST::string thestring = ST::format("{} {}", fString1, fString2);            
```

Comment:
It looks like this changes the behavior slightly. Previously, we would only format the strings together if both were non-null. I think a closer match to the previous behavior would be:
```suggestion
            ST::string_stream ss;
            if (!fString1.empty())
                ss << fString1;
            if (!fString1.empty() && !fString2.empty())
                ss << ' ';
            if (!fString2.empty())
                ss << fString2;
            ST::string thestring = ss.to_string();
```

Reply:
Oh, you're right... it wasn't quite equivalent. Good suggestion!